cypress/integration/rendering/oldShapes.spec.ts

```diff
@@ -0,0 +1,101 @@
+import { imgSnapshotTest } from '../../helpers/util';
+
+const looks = ['classic', 'handDrawn'] as const;
+const directions = ['TB', 'BT', 'LR', 'RL'] as const;
+
+const shapesSet1 = ['text', 'card', 'shadedProcess', 'diamond', 'hexagon'] as const;
+
+const shapesSet2 = ['roundedRect', 'squareRect', 'stateStart', 'stateEnd', 'labelRect'] as const;
+
+const shapesSet3 = ['forkJoin', 'choice', 'note', 'stadium', 'odd'] as const;
+
+const shapesSet4 = ['subroutine', 'cylinder', 'circle', 'doublecircle', 'odd'] as const;
+
+const shapesSet5 = ['anchor', 'lean_right', 'lean_left', 'trapezoid', 'inv_trapezoid'] as const;
```

Comment:
These shapes don't follow the kebab-case convention. 

Also, why did we choose kebab-case? Look is camelCase `handDrawn`. Shouldn't we be consistent?